src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -244,6 +253,13 @@ simple_data_iterator::compute_properties(
       result.class_to_index_map[label] = i++;
     }
   } else {
+    // Make sure the expected labels are unique and sorted.
+    std::sort(expected_class_labels.begin(), expected_class_labels.end());
+    std::vector<std::string>::iterator it;
+    it =
+        std::unique(expected_class_labels.begin(), expected_class_labels.end());
+    expected_class_labels.resize(
+        std::distance(expected_class_labels.begin(), it));
```

Comment:
Why do the labels need to be sorted? The developer can't choose what label is first?

Reply:
I don't know, I follow in implementation in Python side:
https://github.com/apple/turicreate/blob/62ea3f7276ec987ce94956ef52ee217f407d4e33/src/python/turicreate/toolkits/object_detector/object_detector.py#L333

And it seems like in the python side they already sort it and take the unique value of it?
Do I need to do it again in the C++ side?

Reply:
Let's not duplicate that work for now